<?xml version="1.0"?>
<launch>
    <node name="cleaner_robot" pkg="exercise3" type="cleaner_robot" output="screen" />
    <node name="charging_station" pkg="exercise3" type="charging_station" output="screen" />
</launch>